### PR TITLE
Clarify last line in README.md

### PR DESCRIPTION
**Pull request by MichaelClerx**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,9 @@ Documentation on FC can be found [here](https://chaste.cs.ox.ac.uk/trac/wiki/Fun
 
 An ongoing attempt to document the Web Lab and all its interconnected technologies can be viewed in the [weblab docs repository](https://github.com/ModellingWebLab/weblab_docs).
 
+## Linux-only
```
Comment (jonc125) [inline README.md]: It does work on Mac OS though :)

File: README.md [hunk 2]
```diff
@@ -40,7 +43,12 @@ export CFLAGS="-I$HOME/anaconda3/envs/weblab/include"
...
+Because the `weblab_fc` module has Cython components, it needs to be compiled before you can use it.
+Compilation is performed using Python's [`distutils`](https://docs.python.org/3/library/distutils.html) and [`setuptools`](https://setuptools.readthedocs.io/en/latest/), and happens automatically when you install the package using `setup.py`.
+For developers, this can be done using:
+
 ```sh
-pip install .[dev,test]
+pip install -e .[dev,test]
 ```
+
+Note that you'll need to repeat this step after any changes to Cython files (e.g. `.pyx` or `.pxd` files).
```
Comment (jonc125) [inline README.md]: s/./, because these don't automatically get recompiled./

File: README.md (hunk 1 above)
Comment (MichaelClerx) [reply README.md]: That's also linux :D

Comment (jonc125) [reply README.md]: Not really. It's Unix-based, but more like BSD than Linux iirc.

**Comment by MichaelClerx** [comment]
Is this better @jonc125 ?

**Pull request merged**